aws-ecs/ecs.tf

```diff
@@ -0,0 +1,116 @@
+locals {
+  app_name = "example-app"
+  port     = 8080
+}
+
+resource "aws_ecs_cluster" "example_app" {
+  name = "${local.app_name}"
+}
+
+resource "aws_ecs_service" "example_app" {
+  name            = "${local.app_name}"
+  cluster         = "${aws_ecs_cluster.example_app.id}"
+  launch_type     = "FARGATE"
+  task_definition = "${aws_ecs_task_definition.example_app.arn}"
+
+  network_configuration {
+    subnets          = "${var.subnets}"
+    security_groups  = "${list(aws_security_group.example_app.id)}"
+    assign_public_ip = true
```

Comment:
Assigning a public IP instead of the recommended ALB. Also have an example locally that uses ALB instead, but that's more complex, slower and costs more AWS $. Tiny downside though of using a public IP for the demo is that it cannot be used as an `output` because there's not attribute for it, so it has to be looked up manually in the console.